app/models/manageiq/providers/infra_manager/metrics_capture.rb

```diff
@@ -1,5 +1,94 @@
 class ManageIQ::Providers::InfraManager::MetricsCapture < ManageIQ::Providers::BaseManager::MetricsCapture
   def capture_ems_targets(options = {})
-    Metric::Targets.capture_infra_targets([ems], options)
+    emses = [ems]
+    load_infra_targets_data(emses, options)
```

Comment:
Could we change `load_infra_targets_data` to work with only a single ems instead of wrapping one in an array?  Is it called by anyone else that expects to be able to use it with multiple emses?

Would be nice to drop the extra loop [here](https://github.com/ManageIQ/manageiq/pull/19684/files#diff-f10ae13cc3478cdb443b3c2fbc68c0dfR54)